apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Transaction.java

```diff
@@ -95,10 +114,14 @@
 
     String getReferencedTransactionFullHash();
 
+    byte[] referencedTransactionFullHash();
+
     byte[] getSignature();
 
     String getFullHash();
```

Comment:
getFullHashString and getFullHash probably could be better

Reply:
ok

Reply:
Both methods were renamed, pushed.